INSTALL.md

```diff
@@ -0,0 +1,25 @@
+
+## Setup CactusBot:
+```
+git clone https://github.com/cactusdev/cactusbot
+cd cactusbot
+pip3 install -r requirements.txt
+cp config.template.py config.py
+```
+
+Next, open `config.py` with your favorite text editor, and set 
+`USERNAME` and `PASSWORD` to the bot's credentials, and set `CHANNEL` to your channel's name.
+
+# Usage
+
+Start Sepal:
+
+`npm start`
+
+Start CactusAPI:
```

Comment:
This should be before "Start Sepal", right? Also, we should mention "Start RethinkDB" somewhere.